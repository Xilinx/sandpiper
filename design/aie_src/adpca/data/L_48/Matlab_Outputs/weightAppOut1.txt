7.878030837e-02 2.379882894e-02
-1.020196438e+01 1.992976904e+00
-1.246559858e+00 -4.493386745e+00
