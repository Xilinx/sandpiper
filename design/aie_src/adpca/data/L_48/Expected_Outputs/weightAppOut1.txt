7.877719402e-02 2.379913814e-02 
-1.020178127e+01 1.992516518e+00 
-1.246531248e+00 -4.493312836e+00 
